app/controllers/searches_controller.rb

```diff
@@ -3,9 +3,10 @@ class SearchesController < ApplicationController
 
   def show
     return unless params[:query].is_a?(String)
-    @error_msg, @gems = ElasticSearcher.new(params[:query], page: @page).search
+    @error_msg, @elastic_results = ElasticSearcher.new(params[:query], page: @page).search
```

Comment:
Can you explain why this change is needed? :thinking:

Reply:
This was me circumventing an issue to get a working proof of concept 😬

`@gems` originally  is assigned a collection of Hash-like objects like this:

```
<Searchkick::HashWrapper _id="15494" _index="rubygems-development_20240219203755949" _score=248.40211 _type="_doc" description="Ruby on Rails is a full-stack web framework optimized for programmer happiness and sustainable productivity. It encourages beautiful code by favoring convention over configuration." downloads=496344261 id="15494" name="rails" summary="Full-stack web application framework." version="7.1.3">
```

I couldn't figure out how to add the `latest_release_date` property to the HashWrapper (Took a look [here](https://github.com/heyapricot/rubygems.org/blob/0c48ef3d8584e6cc3c8f28e9124f8e7c2be913b2/lib/elastic_searcher.rb#L150) for example)

What I did instead was map the id's from the search result collection, run a new query and use the actual Rubygem objects instead.

Reply:
Yup, AFAIK you can use `version` attribute on returned search result.